documentation/release-summary.md

```diff
@@ -154,107 +259,67 @@ This is the Spendr [Architecture Design](https://github.com/BarbzCodez/Spendr/bl
 
 ### Libraries
 
-- [Material UI](https://mui.com/material-ui/)
-  - We used Material UI to make the components in the front end. We chose it because is an industry standard, it has pretty icons and some of our developers have worked with it.
-  - What we used for the phone scanner
-
-List of Dependencies
+- What we used for the phone scanner???????
```

Comment:
@BarbzCodez We used the EdenAI API instead of a library, so I don't think that needs to go here

```suggestion
```